### PR TITLE
Update sample chapter

### PR DESCRIPTION
**Pull request by theresa-jones**
This PR includes a few minor updates to the starting files for new AsciiDoc repositories.

- Fixing a few xref, etc errors, that appear when the current sample_chapter.asciidoc is added as-is to the build list of a new project.
- Adding the example tarsier.png image file, which is referenced in the sample chapter, so that the generate_new_project script can add it to new projects.
- Adding instructions for building the book in Atlas (requested by a DE)

### CODE REVIEW

File: asciidoc_only/sample_chapter.asciidoc [hunk 1]
```diff
@@ -106,10 +107,33 @@ Finally, here's a sample table:
...
+
+First, go to the Dashboard tab of your book project. Then, look to the right panel, click the "PDF" box, and hit "Build". Atlas will take a moment to generate the pages. A "Download" button will appear - click this, and a PDF will open in your web browseer. This PDF may then be saved locally. 
+
+[TIP]
+====
+If the Build button on the configure tab is grayed out, simply refresh the browser. The Build button should then become active.
+====
+
+==== Adding a Chapter to the Book Build 
+In order to add a new chapter in Atlas, you'll need to make an adjustment to the Cofnigure tab in Atlas.
```
Comment (delfanbaum) [inline asciidoc_only/sample_chapter.asciidoc:129]:
```suggestion
In order to add a new chapter to your book in Atlas, you'll need to make an adjustment to the Configure tab in Atlas. 
```

File: asciidoc_only/sample_chapter.asciidoc [hunk 2]
```diff
@@ -106,10 +107,33 @@ Finally, here's a sample table:
...
+[TIP]
+====
+If the Build button on the configure tab is grayed out, simply refresh the browser. The Build button should then become active.
+====
+
+==== Adding a Chapter to the Book Build 
+In order to add a new chapter in Atlas, you'll need to make an adjustment to the Cofnigure tab in Atlas. 
+
+. Move to the Configure tab in the book project in Atlas. 
+. Scroll down to the middle section of the subsequent page under "Files". Find the chapter you'd like to add, and click the + symbol.
```
Comment (delfanbaum) [inline asciidoc_only/sample_chapter.asciidoc:132]:
```suggestion
. Scroll to the middle section of the page under "Files." Find the chapter you'd like to add and click the + symbol.  
```

File: asciidoc_only/sample_chapter.asciidoc [hunk 3]
```diff
@@ -106,10 +107,33 @@ Finally, here's a sample table:
...
+====
+
+==== Adding a Chapter to the Book Build 
+In order to add a new chapter in Atlas, you'll need to make an adjustment to the Cofnigure tab in Atlas. 
+
+. Move to the Configure tab in the book project in Atlas. 
+. Scroll down to the middle section of the subsequent page under "Files". Find the chapter you'd like to add, and click the + symbol.  
+. This will move the chapter to the "Build Contents" section on the right side of the page. 
+.  Use the three stacked lines to reorder the chapters as desired. 
+. Once your contents list looks correct, scroll down the page to click the "Save Settings" button.
```
Comment (delfanbaum) [inline asciidoc_only/sample_chapter.asciidoc:135]:
```suggestion
. Once your contents list looks correct, scroll down the page to click the "Save Settings."
```

File: asciidoc_only/sample_chapter.asciidoc [hunk 4]
```diff
@@ -106,10 +107,33 @@ Finally, here's a sample table:
...
+
+==== Adding a Chapter to the Book Build 
+In order to add a new chapter in Atlas, you'll need to make an adjustment to the Cofnigure tab in Atlas. 
+
+. Move to the Configure tab in the book project in Atlas. 
+. Scroll down to the middle section of the subsequent page under "Files". Find the chapter you'd like to add, and click the + symbol.  
+. This will move the chapter to the "Build Contents" section on the right side of the page. 
+.  Use the three stacked lines to reorder the chapters as desired. 
+. Once your contents list looks correct, scroll down the page to click the "Save Settings" button. 
+.  Go back to the Dashboard tab, and repeat the steps to create a Book Build as discussed in the previous section.
```
Comment (delfanbaum) [inline asciidoc_only/sample_chapter.asciidoc:136]:
```suggestion
.  Go back to the Dashboard and run a new book build as discussed in the previous section to verify your changes. 
```

File: asciidoc_only/sample_chapter.asciidoc [hunk 5]
```diff
@@ -106,10 +107,33 @@ Finally, here's a sample table:
...
+A helpful feature of Atlas is to create a book build in real time. This build shows the interior design of the book in PDF format, which allows you to see how elements will render on the print page. 
+
+First, go to the Dashboard tab of your book project. Then, look to the right panel, click the "PDF" box, and hit "Build". Atlas will take a moment to generate the pages. A "Download" button will appear - click this, and a PDF will open in your web browseer. This PDF may then be saved locally. 
+
+[TIP]
+====
+If the Build button on the configure tab is grayed out, simply refresh the browser. The Build button should then become active.
+====
+
+==== Adding a Chapter to the Book Build
```
Comment (delfanbaum) [inline asciidoc_only/sample_chapter.asciidoc:128]: Question: would it be worth noting that the same thing can be done in the _atlas.json_ file? Typically, I think authors who write in asciidoc/use atlas will be comfortable editing the json directly also, as an option.

**Review by delfanbaum — CHANGES_REQUESTED** [review]

**Comment by theresa-jones** [comment]
@delfanbaum Thanks! Made the suggested changes, much better without the extra capitalization especially. Good suggestion about editing atlas.json directly, I added a note about that

File: asciidoc_only/sample_chapter.asciidoc (hunk 1 above)
Comment (theresa-jones) [reply asciidoc_only/sample_chapter.asciidoc]: Fixed

File: asciidoc_only/sample_chapter.asciidoc (hunk 3 above)
Comment (theresa-jones) [reply asciidoc_only/sample_chapter.asciidoc]: Fixed.

File: asciidoc_only/sample_chapter.asciidoc (hunk 5 above)
Comment (theresa-jones) [reply asciidoc_only/sample_chapter.asciidoc:133]: Good idea, added this in a Tip element

File: asciidoc_only/sample_chapter.asciidoc (hunk 4 above)
Comment (theresa-jones) [reply asciidoc_only/sample_chapter.asciidoc]: Changed according to the suggestion.

File: asciidoc_only/sample_chapter.asciidoc (hunk 2 above)
Comment (theresa-jones) [reply asciidoc_only/sample_chapter.asciidoc]: Changed according to the suggestion.

File: asciidoc_only/sample_chapter.asciidoc [hunk 6]
```diff
@@ -106,10 +112,39 @@ Finally, here's a sample table:
...
 To include hyperlinks to external sources, add the full URL followed by the anchor text enclosed in single square brackets. This, for example, is a link to the https://www.oreilly.com/[O'Reilly home page]. The bracketed text will become a clickable link in web versions. In print versions, it will appear in the text, followed by the actual URL in parentheses.
+
+=== Creating a Book Build in Atlas 
+A helpful feature of Atlas is to create a book build in real time. This build shows the interior design of the book in PDF format, which allows you to see how elements will render on the print page. 
+
+First, go to the Dashboard tab of your book project. Then, look to the right panel, click the "PDF" box, and hit "Build". Atlas will take a moment to generate the pages. A "Download" button will appear - click this, and a PDF will open in your web browser. This PDF may then be saved locally. 
+
+[TIP]
+====
+If the Build button on the configure tab is grayed out, simply refresh the browser. The Build button should then become active.
```
Comment (nadamsoreilly) [inline asciidoc_only/sample_chapter.asciidoc:130]: I think we should remove "on the configure tab".

**Review by nadamsoreilly — CHANGES_REQUESTED** [review]
Looks good! One small change requested to make an instruction clearer

File: asciidoc_only/sample_chapter.asciidoc (hunk 6 above)
Comment (theresa-jones) [reply asciidoc_only/sample_chapter.asciidoc]: Good catch! I fixed this.

File: asciidoc_only/sample_chapter.asciidoc [hunk 7]
```diff
@@ -47,7 +47,7 @@ print "Hello World"
 ----
 ====
 
-For either type of listing, you have the option of specifying the code language displayed; see https://docs.atlas.oreilly.com/ch12.html#asciidocref for more details.
+For either type of listing, you have the option of specifying the code language displayed; see the Code section of the Atlas user docs here -  https://docs.atlas.oreilly.com/writing_in_asciidoc.html for more details.
```
Comment (delfanbaum) [inline asciidoc_only/sample_chapter.asciidoc]:
```suggestion
For either type of listing, you have the option of specifying the code language displayed; see the Code section of the https://docs.atlas.oreilly.com/writing_in_asciidoc.html[Atlas user docs] for more details.
```

File: asciidoc_only/sample_chapter.asciidoc [hunk 8]
```diff
@@ -106,10 +107,33 @@ Finally, here's a sample table:
...
+Here's a cross-reference to <<example_code_block>>. Here's one to <<example_table>>. Finally, here's one to <<sample_chapter_unique_id>>.
 
 For the book to be valid, ids must be unique across the entire book, have no spaces, and not start with a number.
 
 O'Reilly house style calls for every formal figure, formal table, and formal code example to be preceded by a specific in-text reference. Formal figures and formal tables are created when you add captions to these elements.
 
 To include hyperlinks to external sources, add the full URL followed by the anchor text enclosed in single square brackets. This, for example, is a link to the https://www.oreilly.com/[O'Reilly home page]. The bracketed text will become a clickable link in web versions. In print versions, it will appear in the text, followed by the actual URL in parentheses.
+
+=== Creating a Book Build in Atlas 
+A helpful feature of Atlas is to create a book build in real time. This build shows the interior design of the book in PDF format, which allows you to see how elements will render on the print page.
```
Comment (delfanbaum) [inline asciidoc_only/sample_chapter.asciidoc:124]:
```suggestion
A helpful feature of Atlas is to create a book build in real time. The PDF build shows the interior design of the book in PDF format, which allows you to see how elements will render on the print page, while the EPUB build generates an epub. 
```

File: asciidoc_only/sample_chapter.asciidoc [hunk 9]
```diff
@@ -106,10 +107,33 @@ Finally, here's a sample table:
...
-Here's a cross-reference to <<unique_id_01>>. Here's one to <<unique_id_02>>. Finally, here's one to <<unique_chapter_id_01>>.
+Here's a cross-reference to <<example_code_block>>. Here's one to <<example_table>>. Finally, here's one to <<sample_chapter_unique_id>>.
 
 For the book to be valid, ids must be unique across the entire book, have no spaces, and not start with a number.
 
 O'Reilly house style calls for every formal figure, formal table, and formal code example to be preceded by a specific in-text reference. Formal figures and formal tables are created when you add captions to these elements.
 
 To include hyperlinks to external sources, add the full URL followed by the anchor text enclosed in single square brackets. This, for example, is a link to the https://www.oreilly.com/[O'Reilly home page]. The bracketed text will become a clickable link in web versions. In print versions, it will appear in the text, followed by the actual URL in parentheses.
+
+=== Creating a Book Build in Atlas
```
Comment (delfanbaum) [inline asciidoc_only/sample_chapter.asciidoc:123]:
```suggestion
=== Creating a Book Build in Atlas 

```

(Add an extra space to ensure proper conversion/this is generally good practice for authors)

File: asciidoc_only/sample_chapter.asciidoc [hunk 10]
```diff
@@ -106,10 +107,33 @@ Finally, here's a sample table:
...
 
 O'Reilly house style calls for every formal figure, formal table, and formal code example to be preceded by a specific in-text reference. Formal figures and formal tables are created when you add captions to these elements.
 
 To include hyperlinks to external sources, add the full URL followed by the anchor text enclosed in single square brackets. This, for example, is a link to the https://www.oreilly.com/[O'Reilly home page]. The bracketed text will become a clickable link in web versions. In print versions, it will appear in the text, followed by the actual URL in parentheses.
+
+=== Creating a Book Build in Atlas 
+A helpful feature of Atlas is to create a book build in real time. This build shows the interior design of the book in PDF format, which allows you to see how elements will render on the print page. 
+
+First, go to the Dashboard tab of your book project. Then, look to the right panel, click the "PDF" box, and hit "Build". Atlas will take a moment to generate the pages. A "Download" button will appear - click this, and a PDF will open in your web browseer. This PDF may then be saved locally.
```
Comment (delfanbaum) [inline asciidoc_only/sample_chapter.asciidoc]:
```suggestion
To build a PDF, go to the Dashboard tab of your book project. Then, click the "PDF" checkbox box under "Builds", and hit "Build" Atlas will take a moment to generate the pages. A "Download" button will appear. Click this to open the PDF.
```

(The open-in-browser thing works only in Chrome, I think; we can be less explicit here -- we ought to assume our authors will be familiar with viewing/downloading PDFs)

File: asciidoc_only/sample_chapter.asciidoc (hunk 1 above)
Comment (delfanbaum) [inline asciidoc_only/sample_chapter.asciidoc]:
```suggestion
In order to add a new chapter in Atlas, you'll need to add the file using the Configure tab in your book's repository page. 
```

File: asciidoc_only/sample_chapter.asciidoc (hunk 2 above)
Comment (delfanbaum) [inline asciidoc_only/sample_chapter.asciidoc]:
```suggestion
. Scroll down to the middle section of the subsequent page under "Files." Find the chapter you'd like to add, and click the + symbol.  
```

**Review by delfanbaum — COMMENTED** [review]
I left some suggestions; a few I insist upon.

**Review by delfanbaum — APPROVED** [review]
Okay, looks good! Added one optional suggestion.

**Review by nadamsoreilly — APPROVED** [review]

**Pull request merged**